addon/controllers/messages_base.js

```diff
@@ -35,9 +41,9 @@ export default Ember.Controller.extend({
 
   messages: Ember.computed("allMessages.[]", "offer", "item", function() {
     var messages = this.get("allMessages");
-    messages = this.get("isItemThread") ?
-      messages.filterBy("itemId", this.get("item.id")) :
-      messages
+    messages = this.get("isItemThread")
```

Comment:
Syntax: better to use `if { } else {}` for multi-line statements. Ternary is useful for single line.
cc @patrixr for an opinion